React/Views/RCTViewManager.m

```diff
@@ -82,6 +82,19 @@ - (RCTViewManagerUIBlock)uiBlockToAmendWithShadowViewRegistry:(RCTSparseArray *)
 RCT_REMAP_VIEW_PROPERTY(shadowOpacity, layer.shadowOpacity, CGFloat)
 RCT_REMAP_VIEW_PROPERTY(shadowRadius, layer.shadowRadius, CGFloat)
 RCT_REMAP_VIEW_PROPERTY(transformMatrix, layer.transform, CATransform3D)
+RCT_CUSTOM_VIEW_PROPERTY(rotation, CGFloat, RCTView)
+{
+  if(json)
+  {
+    CGFloat angle = [RCTConvert CGFloat:json];
```

Comment:
Personally I think it would be nice to take radians by default and provide `degrees()` and `radians()` functions in JavaScript to do the appropriate conversions